changelogs/unreleased/gh-6833-encode-body-http-request.md

```diff
@@ -0,0 +1,6 @@
+## feature/lua/http
```

Comment:
Regarding the documentation request for encoding (in the commit message):

* It mentions encoding of a Lua table, but the patch actually handles `nil`, `string`, `table`, `userdata`, `cdata` (only string was handled before, maybe also `nil`).
* There is no word about yaml and msgpack encoding, but the patch provides them.
* There is no word about API to set user's encoder, but it is planned to be a public API AFAIR.

I guess you updated the patch but forgot about the documentation request.